packages/orbit-components/src/Skeleton/README.md

```diff
@@ -49,7 +49,7 @@ or you can pick one from our predefined presets:
 | rowHeight       | `number`              | `21px`             | Set height for rect elements                     |
 | rowOffset       | `number`              | `20px`             | Set offset between rect elements                 |
 | rows            | `number`              |                    | Number of rect elements                          |
-| title           | `string`              | `loading`          | Add text for svg accessible name element         |
+| title           | `string`              |                    | Accessible text for screen readers               |
```

Comment:
I wonder whether we should consider this a breaking change.. in the end, nothing is breaking, as we're handling the scenario when `title` is not set.

Reply:
I was thinking about this and you are right. But then also, if users didn't provide the title prop because they were counting with the default value which will now be removed, that's a breaking change, no?

But anyway, I followed [what Daniel did in the Seat component](https://github.com/kiwicom/orbit/commit/290bf2100be4686f499c0bfa173e89aea2bba96d) where he marked such scenario as a breaking change  .

So what do you suggest, not mark it as breaking change and just add the codemod to migration guide? Or not even create a codemod? 🤔

Reply:
Yeah.. I also believe that strictly speaking, it's a breaking change. I'll raise this topic internally.

Reply:
As agreed offline, let's keep the breaking change.